core/src/syscall/precompiles/sha256/compress/trace.rs

```diff
@@ -134,9 +134,17 @@ impl<F: PrimeField> MachineAir<F> for ShaCompressChip {
                 let e_not_and_g = cols.e_not_and_g.populate(output, e_not, g);
                 let ch = cols.ch.populate(output, e_and_f, e_not_and_g);
 
-                let temp1 = cols
-                    .temp1
-                    .populate(output, h, s1, ch, event.w[j], SHA_COMPRESS_K[j]);
+                // TODO: This is a hack to avoid calling Add5Operation::populate. We currently don't
```

Comment:
nit can you leave the old "correct" code in the comment so people who what it should be fixed to?

Reply:
good point! added.